Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -253,6 +282,9 @@ static ErrorOr<void> decode_iff_chunks(ILBMLoadingContext& context)
             context.state = ILBMLoadingContext::State::BitmapDecoded;
         } else if (chunk.type == FourCC("CRNG")) {
             dbgln_if(ILBM_DEBUG, "Chunk:CRNG");
+        } else if (chunk.type == FourCC("CAMG")) {
+            context.viewport_mode = *bit_cast<BigEndian<u32> const*>(chunk.data.data());
```

Comment:
this kind of type punning even with bit_cast without regard for alignment is a bit spooky. Can we use `AK::ByteReader` for this?

Reply:
I updated the code to use `ByteReader`: let me know if it's better.

Reply:
I'll think about it :) But I'd rather finish HAM+EHB support first :)